main.js

```diff
@@ -14,15 +14,17 @@ function createWindow () {
 
   // and load the index.html of the app.
   mainWindow.loadFile('index.html')
-
-  // Open the DevTools.
-  // mainWindow.webContents.openDevTools()
 }
 
 // This method will be called when Electron has finished
 // initialization and is ready to create browser windows.
 // Some APIs can only be used after this event occurs.
 app.whenReady().then(() => {
+  // Prepare for child processes to indicate test results.
+  ipcMain.on('test-done', () => finishTest(true))
```

Comment:
maybe the `test-done` ipc could take an argument for pass/fail? i guess it's redundant with `process.exit(1)` in the renderer...

Reply:
The redundancy was my thinking, too. Since we're trying to KISS I didn't want to have multiple code paths for this muddling the template.

Reply:
OTOH now that I reread it, I think the current draft may be less intuitive / cause more confusion than your suggestion. Let's add it even if it is redundant.

Reply:
Added in 544712d14681697e4c155e09ff509e7461216e7a